### ISSUE TITLE
List of pending issues from comments on spreadsheet

### ISSUE THREAD

**Issue by drozas** [drozas/cecan#114]
Waiting for a reply. Transform into issues if necessary:

Please, find below a summary of the changes carried out as from your comments. Those on bold require changes from my side, but are pending on an editorial decision.

> Main logo

This is pending on receiving the new logo from the designer (as from the previous e-mails). +info at https://github.com/drozas/cecan/issues/77

> Capitalisation of CECAN in those menu entries

This was already done.

> Font size in some nodes

This is editorial content

> I prefer no rotating pictures, just one is fine

This was already done.

> Spell out the co-funders in about (and add new text)

This is done in the new version of the block.

> The about CECAN text needs imroving + home page should include panel with twitter

This has been updated with the new text you provided in the new version

> - the phrase - do you want to know more about us - is odd, suggest remove this button

This is done in the new version

> - picture not appropriate:

This was the image chosen (see https://github.com/drozas/cecan/issues/70). If you want me to replace it, please let me know which image would you like to use. Please, consider previous comments on the dimensions (current picture is 1366 x 600).

> The buttons to the right of main image are odd

These buttons possess the style of the theme chosen by the team (https://www.drupal.org/project/icemagic). If you want me to perform some changes (e.g. colour, font size, etc., could you please send me a design/ideas of how you expect them to be?

> Founder logos

This is hopefully done now (see previous e-mail) :-)

> - 'researching' and 'what we do' section probably could be merged, and better text prvided:

This is editorial content, not development involved.

> - I thought the calendar was going to be removed - stil visible at bottom of first page:

Removed in new version.

> Case studies logos

These logos are part of an image field. The agreed dimensions were 200x150 (see https://github.com/drozas/cecan/issues/61). You need to either upload logos which fulfill these dimensions, or letting me know which new dimensions would you like to employ.  I think Julia carried out the changes already.

> Increase the number of events

I increased the number of events per view from 3 to 6 already for all the views. If you want to increase it even more, please let me know how many exactly.

> Increase the number of blogposts

- I maintained 6 blogs per page, for coherence as with the events. If you want me to increase the number more,  please let me know how many exactly.

> Blog images

With regards to the sizes of the image, this is under editorial control (see https://github.com/drozas/cecan/issues/84). I provided a set of instructions and templates for this via e-mail, but please let me know if there is still anything unclear about this. I think Julia has already taken care of this.

> Video titles

With regards to the size and lining up, I added a padding and center the titles. There is a small issue with this, which is that certain titles are extremely long (e.g. http://drozas.com/videos/context-mechanism-association-mastering-a-key-heuristic-realist-evaluation-for-innovating), while others are very short (e.g. http://drozas.com/videos/cecan-seminar-dependency-modelling). Thus, I added the maximum amount of padding so it looked equilibrated. There are possible solutions to this:

    A: Keep it as it is
    B: A possible alternative solution for this is doing as in the case studies (http://drozas.com/case-studies), in which the title are automatically crop after certain amount of characters (e.g. 300). Could you please let me know which solution would you like me to employ. I can increase the size, that is not problem.
    C: Increase the padding even more, but then those with a short title will look strange

There was also a request to increase the size of the titles (but this would affect the previous solution). Should I increase it? (after considering the previous issue)

> SE England text.

This is editorial content.

**Comment by drozas** [comment]
Keeping open. Will create new issues from here if requested by the editorial team.

**Comment by drozas** [comment]
Ḿore issues: to see with Julia

Hi all,


Apologies that I've taken so long to engage with this, I've been ill for most of last week.  I think that the new image of wind turbines is much better on the front page.


An ongoing issue for me (that I think is a development issue) is the lack of an index or other way of quickly accessing the information you need, on several of the pages. This is the case on:

 - About us / Advisory Board

-  Events

 - Blogs

 - News

 - Videos

...For instance, if I wanted to quickly find the seminar we had, run by Collingwood Environmental Planning, I have to slowly scroll through the first page of events / archived events, and then manually dig that link out from the second page. I don't really mind how this is improved (adding a 'contents list' at the top of the web pages / tagging content with certain tags / whatever) just so long as it become easier to quickly access the specific page I'm after..


Another minor development issue: I think that the full name of the case studies should be displayed on that page (all 4 case studies peter out into'...' after a certain number of characters)


A content suggestion for the Centre Managers: the CECAN fellows should have the full name of their fellowships displayed next to their picture (not just their names).


I'll send over more thoughts as they occur to me.


All the best,


Ben

**Comment by drozas** [comment]
I agree with Ben that the front page is much better with the new image.  And generally I like the ‘look and feel’ of the new design.

 

Here are some further suggestions and comments:

 

Many visitors will go only to the front page and never venture further.  That’s why recent site designs have adopted the ‘very long front page’ idea (see for example, the University of Surrey’s front page, or the example I mentioned before, https://www.instituteforgovernment.org.uk/) In contrast, our new front page has very little information on it.  Is it worth considering including recent news/event/blog/etc. items on the front page, like those two examples do?

 

The CECAN logo in the top menu still looks odd, because of its white background.  Can we ask Linda for advice on how to make a version that looks OK on a dark background?

 

I am finding it hard to discover the logic behind the buttons overlaying the wind turbine image – what is the significance of the background colours, and why are they placed in rows of different lengths?  Even if they are left as they are, they would look better with a margin of clear space (i.e. the image showing through) around each one.  The buttons would also perhaps look better is they had a shadow applied.

 

The links in the footer (e.g. ‘Drupal’) are blue on a blue background, so they are illegible.

 

Nigel

**Comment by drozas** [comment]
Pete:
Thanks for all these.

 

On the points I can help with now…

 

Buttons to the right on main image – I would expect them all in one line arcoss the bottom of the image, as in the image attached (this is v rough but gives the idea)

Events and blogs – I would like 10 or 15 per pages with each one smaller – a bit more like the videos page in overall style

Video titles – I think they are ok as is, and to improve them is an editorial decision on us – most can be made shorter by us.

 

P

**Comment by drozas** [comment]
![buttons1](https://user-images.githubusercontent.com/1962473/32042419-b74ccd08-ba37-11e7-81cb-a38ed639522f.png)

**Comment by drozas** [comment]
Thanks Pete, Julia,

I have compiled the latest comments from Nigel, Ben and yours (see for more detail https://github.com/drozas/cecan/issues/114) and try to mix them into a new set of requirements. I have, however, some questions regarding the changes before applying them:

> P: Buttons to the right on main image – I would expect them all in one line arcoss the bottom of the image, as in the image attached (this is v rough but gives the idea)

I can move the buttons down. Nigel also made some comments regarding this (see https://github.com/drozas/cecan/issues/114#issuecomment-338712868). Do you think placing them down and changing a bit the style, as Nigel suggested, would made the trick for everyone?

> N: Many visitors will go only to the front page and never venture further. That’s why recent site designs have adopted the ‘very long front page’ idea (see for example, the University of Surrey’s front page, or the example I mentioned before, https://www.instituteforgovernment.org.uk/) In contrast, our new front page has very little information on it. Is it worth considering including recent news/event/blog/etc. items on the front page, like those two examples do?
Should we create new blocks for the homepage? I think we could place 2 or 3 blocks in a new section, between the "welcome" and the logos. If so, which ones (e.g. news and blog posts, with the latest 3). I can create also a block for forthcoming events. However, the problem with this, is that at times might not be three, so it might look a bit strange.

> P: Video titles – I think they are ok as is, and to improve them is an editorial decision on us – most can be made shorter by us.
> B: Another minor development issue: I think that the full name of the case studies should be displayed on that page (all 4 case studies peter out into'...' after a certain number of characters)

Regarding this comment from Ben for the case studies, the issue is similar to that of the videos. The approach in this block was to have a maximum amount of characters, so it does not happen as in the old videos (http://cecan.ac.uk/news/videos). I can force a padding, but then we would have a similar issue as in the current videos section. Do you think it would be better to solve this from a content perspective (removing the "CECAN Case Study" for the four nodes, which is not adding extra information?

>B: A content suggestion for the Centre Managers: the CECAN fellows should have the full name of their fellowships displayed next to their picture (not just their names).

The content type for cecan fellows does not currently have a "position field". Should I add a new field for this and change the template to display it under the name? If so, this will require editorial work to add this position into all of the current nodes.

> N: The links in the footer (e.g. ‘Drupal’) are blue on a blue background, so they are illegible.

I will change this.

> B: An ongoing issue for me (that I think is a development issue) is the lack of an index or other way of quickly accessing the information you need, on several of the pages. This is the case on:

    About us / Advisory Board

    Events

    Blogs

    News

    Videos

> ...For instance, if I wanted to quickly find the seminar we had, run by Collingwood Environmental Planning, I have to slowly scroll through the first page of events / archived events, and then manually dig that link out from the second page. I don't really mind how this is improved (adding a 'contents list' at the top of the web pages / tagging content with certain tags / whatever) just so long as it become easier to quickly access the specific page I'm after..
I see the point of trying to make the navigation easier of this comment, but I am not really sure how this is expected to be solved. The blogs, for example, do currently have tags and they are displayed in a block in the main page (they currently do not appear because no tags have been added - I removed the testing ones as suggested by Julia). We could create similar tags for each of the content types, for example, but I think it would be a bit strange in terms of presentation. Another possibility to solve Ben's suggestion with an automatised approach would be to create a block with the "Latest updates", including the last 3 or 4 nodes of any content (e.g. blog post, event, etc.). Alternatively, this could be "manually" done by creating a block in which editors manually edit the text to "highlight" certain contents. However, this would require to be constantly updated. Any thoughts on this?

>P: Events and blogs – I would like 10 or 15 per pages with each one smaller – a bit more like the videos page in overall style

I have increased the number of blog posts to 15: http://drozas.com/blog . Regarding the changes in the style, there is an issue with respect to the current architecture. A blog post is currently composed of a title, body and certain categories. To create something similar to the current videos (blog title + blog thumbnail) I would need to include a new image field in the blog content type (current pictures are just part of the body), in a similar way as in the current case studies. This would require, however, editorial work to select a thumbnail for each of these blog posts, and upload it into the new field. Would you want me to follow this approach?

I think this hopefully covers all of the issues from the comments, but if I am missing something please do not hesitate to let me know.

Best,

David

**Comment by drozas** [comment]
After final meetings, marking this as closed (rest of issues from Nigel, Pete and Julia gather in previous or new issues).
